app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index2:index - 1]
+else:
+    filename = "file.txt"
+
+    if "-d" in argv:
+        index = 0
+
+        for _ in range(len(argv) - 1):
+            index += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index:len(argv)]
```

Comment:
here we can replace the manual search for the index of the arguments with the built-in `index` method.
```suggestion
filename = ""
directories = []

if "-f" in argv:
    index = argv.index("-f")
    filename = argv[index + 1]
else:
    filename = "file.txt"

if "-d" in argv:
    index = argv.index("-d")
    directories = argv[index + 1:]
```